locals.pkr.hcl

```diff
@@ -0,0 +1,32 @@
+locals {
+  now_unix_timestamp    = formatdate("YYYYMMDDhhmmss", timestamp())
+  agent                 = format("%s-%s", var.agent_os_type, var.agent_os_version)
+  agent_os_version_safe = replace(var.agent_os_version, ".", "_")
+  image_name            = format("jenkins-agent-%s-%s", var.agent_os_type, var.agent_os_version)
+  aws_instance_type = {
+    "amd64" = "t3.large"  # 2 CPU / 1 GB / $0.0104
+    "arm64" = "t4g.micro" # 2 CPU / 1 GB / $0.0084
+  }
+  windows_winrm_user = {
+    "azure-arm"  = "packer"
+    "amazon-ebs" = "Administrator" # In AWS EC2, WinRM super admin must be the "Administrator" account
+    "docker"     = "packer"
+  }
+  azure_vm_size        = "Standard_D4s_v3" # Huge size requires - avoid https:#docs.microsoft.com/en-us/azure/virtual-machines/linux/image-builder-troubleshoot#sysprep-timing and avoid full disk (DS2v2 only have 14 Gb SSD for system)
```

Comment:
```suggestion
  azure_vm_size        = "Standard_D4s_v3" # 4 CPU / 16 GB / Huge size required to avoid https:#docs.microsoft.com/en-us/azure/virtual-machines/linux/image-builder-troubleshoot#sysprep-timing and avoid full disk (DS2v2 only have 14 Gb SSD for system)
```